package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ramda.github.io",
-  "version": "0.0.0",
+  "version": "0.19.1",
```

Comment:
If we're to use this field, we should update it automatically as part of the workflow. `"0.0.0"` is better than an outdated value. ;)